frontend/packages/topology/src/actions/TopologyActions.tsx

```diff
@@ -13,7 +13,7 @@ const TopologyActions: React.FC<TopologyActionsProps> = ({ element }) => {
     const resource = getResource(element);
     const { csvName } = element.getData()?.data ?? {};
     return {
-      'topology-actions': element,
+      'topology-context-actions': { element },
```

Comment:
Why did you remove `topology-actions`?

Reply:
Changed to `topology-context-actions` because many of the add to application actions in `topology-actions` were not coming from dynamic plugins, but hardcoded in add-resources-menu-utils.ts which I removed in this PR.

Reply:
we use `topology-actions` for helm actions as well. So this will break that?

Reply:
Added helm actions to helm's topology provider.

Reply:
it's not just helm topolofy-actions context is being used in other places as well and the provider doesn't expect `{ element }` in the argument.
Please add `topology-context-actions` as a seperate context

Reply:
Oh, did not know that a single consumer could use multiple contexts : p updated.